kge/model/kge_model.py

```diff
@@ -393,23 +398,24 @@ def load_from_checkpoint(
         appended to) in the checkpoint's folder.
 
         """
+        if type(checkpoint) is str:
+            checkpoint = load_checkpoint(checkpoint, device=device)
 
-        checkpoint = torch.load(filename, map_location=device)
+        config = Config.load_from(checkpoint["config"], config)
```

Comment:
This should just be: config.load_from(checkpoint, override=config) or so. Which keys to use is up to config / dataset / train / etc.

Reply:
now called create_from